js/package-readme-template.md

```diff
@@ -0,0 +1,40 @@
+[//]: # (Replace all occurrences of 'your_package', 'your_package_desription' and 'your_username'. Pick one of the Version informartion texts. Leave the comment as it is used to auto-update that part.)
+
+# your_package
+
+#### your_package_desription
+
+Status and support
+
+- &#x2714; / &#x2716; stable / not stable
+- &#x2714; / &#x2716; supported / not supported
+- &#x2714; / &#x2716; ongoing development / no ongoing development
+
+<!--- Version informartion -->
+*You are viewing the README of the development version. There are no releases yet.*
+
+*You are viewing the README of version [v1.0.0](https://github.com/your_username/your_package/releases/tag/v1.0.0). You can find other releases [here](https://github.com/your_username/your_package/releases).*
+
+*You are viewing the README of the development version. You can find the README of the latest release (v1.0.0) [here](https://github.com/your_username//your_package/releases/tag/v1.0.0).*
+<!--- Version informartion end -->
+
+| Branch | Status |
+| ------ | ------ |
+| Release | [![Build Status](https://travis-ci.org/your_username/your_package.svg?branch=release)](https://travis-ci.org/your_username/your_package)   [![Coverage Status](https://coveralls.io/repos/github/your_username/your_package/badge.svg?branch=release)](https://coveralls.io/github/your_username/your_package?branch=release)   [![npm version](https://badge.fury.io/js/%40your_username%2Fyour_package.svg)](https://badge.fury.io/js/%40your_username%2Fyour_package)   [![Total Downloads](https://img.shields.io/npm/dw/@your_username/your_package.svg)](https://www.npmjs.com/package/@your_username/your_package) |
+| Development | [![Build Status](https://travis-ci.org/your_username/your_package.svg?branch=master)](https://travis-ci.org/your_username/your_package)   [![Coverage Status](https://coveralls.io/repos/github/your_username/your_package/badge.svg?branch=master)](https://coveralls.io/github/your_username/your_package?branch=master) |
+
+[//]: # (The following lines should be added after `Usage` chapter replacing the rest of default README.)
+
+## Contribution and feedback
+
+This project is built around known use-cases. If have one that isn't covered don't hesitate to open an issue and start a discussion.
+
+Bug reports and pull requests are welcome on GitHub at https://github.com/your_username/your_package. This project is intended to be a safe, welcoming space for collaboration, and contributors are expected to adhere to the [Contributor Covenant](http://contributor-covenant.org) code of conduct.
+
+## Conventions
+
+This project follows [C-Hive guides](https://github.com/c-hive/guides) for code style, way of working and other development concerns.
+
+## License
+
+The gem is available as open source under the terms of the [MIT License](http://opensource.org/licenses/MIT).
```

Comment:
Shouldn't we write it in a bit more abstract way? Instead of `gem`, I'd propose using `your_repo_type` or something that expresses the general usage of this line.

Reply:
Good point! Let's call it `package` here.